test/components/plots/rectanglePlotTests.ts

```diff
@@ -14,9 +14,9 @@ describe("Plots", () => {
       { x: 4, y: 4, x2: 5, y2: 5 }
     ];
 
-    var VERIFY_CELLS = (cells: D3.Selection) => {
+    var VERIFY_CELLS = (cells: d3.Selection<any>) => {
       assert.strictEqual(cells[0].length, 5);
-      cells.each(function(d: D3.Selection, i: number) {
+      cells.each(function(d: any, i: number) {
```

Comment:
Yes, because the `d` there is the data value, not the `Selection`. The previous author made a mistake.